Common/TableProducer/trackextension.cxx

```diff
@@ -118,7 +113,7 @@ struct TrackExtension {
   }
   PROCESS_SWITCH(TrackExtension, processRun2, "Process Run2 track extension task", true);
 
-  void processRun3(aod::FullTracks const& tracks, aod::Collisions const&)
+  void processRun3(aod::FullTracks const& tracks, aod::Collisions const&, aod::BCsWithTimestamps const&)
```

Comment:
Is it enough to subscribe to Tracks instead of FullTracks?

Reply:
For Run3 yes! Tracks should be enough!